cfy_manager/utils/systemd.py

```diff
@@ -73,12 +73,13 @@ def configure(self, service_name, render=True, tmpfiles=False):
 
         self.systemctl('daemon-reload')
 
-    def remove(self, service_name):
+    def remove(self, service_name, service_file=True):
         """Stop and disable the service, and then delete its data
         """
         self.stop(service_name, ignore_failure=True)
         self.disable(service_name, ignore_failure=True)
-        remove(self.get_service_file_path(service_name))
+        if service_file:
```

Comment:
Ditto - let's add a comment to explain when we don't want to delete the service file